microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -71,7 +71,7 @@
         ),
         OsPackageDependencies(
             matcher=lambda x: isinstance(x, Suse)
-            and parse_version(x.information.release) == "15.5.0",
+            and bool(parse_version(x.information.release) == "15.5.0"),
```

Comment:
It doesn't need the `bool` conversion.

Reply:
![image](https://github.com/user-attachments/assets/21551e69-2f48-4e51-96a9-7efc7bae82cf)

Reply:
parse_version can apparently return None; casting to bool is still safe since None == "15.5.0" will return False (in the event the release info is garbage).

I'm pretty sure that situation would break code elsewhere before we hit this check, so this is really just to satisfy the type checker.